source/conf.py

```diff
@@ -0,0 +1,56 @@
+# Configuration file for the Sphinx documentation builder.
```

Comment:
I can't tell if this file needs copyright / license headers.

Reply:
This file is a setting file and adds no creative content, so I suppose it is not mandatory. - But I am not a copyright expert.
However I found a shorter form of the licence and I can add this to this file: http://www.apache.org/foundation/license-faq.html#Apply-My-Software
WDYT?

Reply:
Changed in 9e559f7